csharp_dotnetcore/3.Welcome-Users/State/WelcomeUserState.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WelcomeUser.State
+{
+    /// <summary>
+    /// Represent state per user in a conversation.
```

Comment:
Consider making this a more complete sentence. As it sits now, it's doesn't really say very much. If I didn't already know what was going on here, this comment wouldn't have helped me.